### PR TITLE
(fix) stock management table not filtering on combox and improving stock UI

### PR DESCRIPTION
**Pull request by samsonmwanzia**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

### CODE REVIEW

**Comment by jabahum** [comment]
@samsonmwanzia lets fix the build errors

File: src/core/components/card/metrics-card.scss [hunk 1]
```diff
@@ -61,6 +61,10 @@
   & > p {
     margin: 0;
   }
+  background-color: #c2d4d4;
```
Comment (donaldkibet) [inline src/core/components/card/metrics-card.scss:64]: We should be using color tokens and spacing  from carbon
e.g 
```scss
margin-botton: layout.$spacing-05;
color: $color-gray-70;
```

the above are just example of how it should be done.

**Review by donaldkibet — CHANGES_REQUESTED** [review]

**Comment by makombe** [comment]
Also, resolve the conflicts @samsonmwanzia

**Comment by hadijahkyampeire** [comment]
@samsonmwanzia @donaldkibet @jabahum I am going to close this as well, I think it was fixed already.

**Pull request closed without merging**